inscriptions/ISic1497.xml

```diff
@@ -181,7 +181,7 @@
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#ME">
                 <ab>
                     <lb n="1"/>
-                    <supplied reason="undefined" evidence="previouseditor"><expan><abbr>C</abbr><ex>aio</ex></expan> <g type="interpunct">·</g></supplied> Terentio <g type="interpunct">·</g> Symphoro
+                    <supplied reason="undefined" evidence="previouseditor"><expan><abbr>C</abbr><ex>aio</ex></expan></supplied> <supplied reason="undefined" evidence="previouseditor"><g type="interpunct">·</g></supplied> Terentio <g type="interpunct">·</g> Symphoro
```

Comment:
Are we sure the interpunct is supplied by the editor?

Reply:
I have to check. That's what indicated in the text before my changes...